### ISSUE TITLE
Make localization packages available on package managers

### ISSUE THREAD

**Issue by RinkAttendant6** [tinymce/tinymce#3616]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
Localization packages available on https://www.tinymce.com/download/language-packages/ do not appear to exist on package maangers (npm, Bower, NuGet).

**What is the expected behavior?**
It would be nice to be able to install the packages using a package manager, either as a subdirectory of the tinymce package or as a separate package altogether, instead of manually downloading them and placing them somewhere.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
Not applicable

**Comment by Charius** [comment]
Agreed with @RinkAttendant6 This feature would be great. But I think that langauge-packs should be in tinymce-dist repo, don't they?

**Comment by RinkAttendant6** [comment]
I see that this popped up not too long ago: https://github.com/mklkj/tinymce-i18n

But it would be nice to get something more official!

**Comment by lomby92** [comment]
This functionality will be very useful, especially for those who use TinyMCE with Webpack. I hope it will be realized as soon as possible

**Comment by Steeve55** [comment]
Agreed for this features. It would be very helpful.

**Comment by RinkAttendant6** [comment]
The issue is still applicable

**Comment by Steeve55** [comment]
I agree, still applicable.
